data/wanderer/wanderers start.txt

```diff
@@ -36,7 +36,7 @@ mission "First Contact: Wanderer"
 	source
 		attributes "wanderer"
 	to offer
-		has "First Contact: Hai: offered"
+		has "discovered hai"
```

Comment:
Technically the player could have "discovered" hai space but not actually gone through the first contact, as it's possible to just keep deferring the first contact missions. You should instead check if either of the first contact missions have been finished.
```suggestion
		or
			has "First Contact: Hai: offered"
			has "First Contact: Unfettered: offered"
```